### DISCUSSION TITLE
compiling game

### DISCUSSION THREAD

**Discussion by swag21lol** [KadeArchive/Kade-Engine#421] [Q&A]
i dont know if its just me but, whenever i try to install discord_rpc it says its already installed and than goes on and updates it, and it doesnt update it just sits there, and without it i cant mod, is there an easy fix?

**Comment by puyoxyz** [comment]
are you installing it with `haxelib install discord_rpc`? if yes, thats not the right command, look at BUILDING.md (`haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc`)

**Reply by swag21lol** [reply]
didnt work, its been hours and no update
